docs/terms/evaluation_parameter.md

```diff
@@ -1,15 +1,132 @@
 ---
 id: evaluation_parameter
 title: Evaluation Parameter
-hoverText: A parameter used during Validation of an Expectation to reference simple expressions or previousl generated metrics.
+hoverText: A connector to store and retrieve information about parameters used during Validation of an Expectation which reference simple expressions or previously generated metrics.
 ---
+
 import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
+import TechnicalTag from '../term_tags/_tag.mdx';
+import CreateHeader from '/docs/images/universal_map/_um_create_header.mdx';
+import ValidateHeader from '/docs/images/universal_map/_um_validate_header.mdx';
+
+
+<UniversalMap setup='inactive' connect='inactive' create='active' validate='active'/> 
+
+## Overview
+
+### Definition
+
+An Evaluation Parameter is a connector to store and retrieve information about parameters used during <TechnicalTag relative="../" tag="validation" text="Validation" /> of an <TechnicalTag relative="../" tag="expectation" text="Expectation" /> which reference simple expressions or previously generated metrics.
+
+### Features and promises
+
+You can use Evaluation Parameters to configure Expectations to use dynamic values, such as a value from a previous step in a pipeline or a date relative to today.  Evaluation Parameters can be simple expressions such as math expressions or the current date, or reference Metrics generated from a previous Validation run.  During interactive development, you can even provide a temporary value that should be used during the initial evaluation of the Expectation.
+
+### Relationship to other objects
+
+Evaluation Parameters are used in Expectations when Validating data.  <TechnicalTag relative="../" tag="checkpoint" text="Checkpoints" /> use <TechnicalTag relative="../" tag="validation_action" text="Action" /> to store Evaluation Parameters in the <TechnicalTag relative="../" tag="evaluation_parameter_store" text="Evaluation Parameter Store" />.
+
+## Use cases
+
+<CreateHeader/>
+
+When creating expectations through the interactive method, Evaluation Parameters can be provided a temporary initial value.  This will allow you to test Expectations that are meant to rely on values from previous Validation runs before you have actually used them to Validate data.  Say you have created a Pandas dataframe called `my_df` that references a CSV that contains ten rows of data.  (To do so in the interactive process for creating Expectations, you would use the `great_expectations.read_csv()` method).  You want to create an expression that asserts that the row count for each Validation remains the same as the previous `upstream_row_count`, but since there is no previous `upstream_row_count` you need to provide a value that matches what the Expectation you are creating will find.
```

Comment:
Since this is the `use cases` section, I would clarify that this use case is `Validate an Expectation against a previous run of the same Suite`, or something along those lines. In contrast, the URN approach allows for validating an expectation against a result in a different suite (useful in part since this means it can be generated from a different data source!)

Reply:
I've added reference and an example of how to use an URN to pull values from an Evaluation Parameter Store.